### ISSUE TITLE
Icon request: icon-photofiltre

### ISSUE THREAD

**Issue by Berkmann18** [ForkAwesome/Fork-Awesome#105]
Alongside Ps, Ai and GIMP, I think that having Photofiltre in the list would be amazing.

![pfsx](https://images.ecosia.org/Xt7sXkG-W_SR1NSZZloO25axja8=/0x390/smart/https%3A%2F%2F3.bp.blogspot.com%2F-bP1uxUpcjdc%2FViaE1JArnSI%2FAAAAAAAAAUg%2FANu5DLmrpw8%2Fs1600%2FPhotoFiltreStudio.png)

**Comment by xuv** [comment]
Is this the official logo? Is there any black and white version somewhere?
Is this their website: http://www.photofiltre-studio.com/ ?

**Comment by Berkmann18** [comment]
@xuv Yes, it's one of the official ones, I believe its the small size version of their software.
Yes, it looks like it's there website.

![b&w](https://images.ecosia.org/HyosIFdvw7u4KQnH60oGHXjgaVo=/0x390/smart/https%3A%2F%2Fopenclipart.org%2Fimage%2F2400px%2Fsvg_to_png%2F229275%2F1444309279.png)
